.github/renovate.json5

```diff
@@ -1,93 +1,81 @@
 {
-  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
   extends: [
-    ':combinePatchMinorReleases',
-    ':separateMajorReleases',
-    ':semanticCommits',
-    ':semanticPrefixChore',
-    ':semanticCommitScopeDisabled',
-    ':updateNotScheduled',
-    ':automergeRequireAllStatusChecks',
-    ':ignoreUnstable',
-    ':gitSignOff',
     'group:monorepos',
     'helpers:disableTypesNodeMajor'
   ],
+
+  labels: ['dependencies'],
+  rangeStrategy: 'bump',
+  prHourlyLimit: 2,
+  prConcurrentLimit: 20,
+  updateNotScheduled: true,
+  commitBody: 'Signed-off-by: {{{gitAuthor}}}',
   ignorePaths: [
     '**/node_modules/**',
     '**/__tests__/**',
     '**/test/**'
   ],
-  force: {
-    constraints: {
-      npm: '>=7'
-    },
-  },
-  labels: [
-    'dependencies'
-  ],
+
+  // Semantic Commits
+  semanticCommits: 'enabled',
+  semanticCommitType: 'chore',
+  semanticCommitScope: null, // Null is valid!
+
+  // Dependency Dashboard
   dependencyDashboard: true,
-  dependencyDashboardApproval: false,
   dependencyDashboardAutoclose: true,
+
+  // Lock File Maintenance
   lockFileMaintenance: {
     enabled: true,
     automerge: true,
-    extends: [
-      'schedule:earlyMondays'
-    ]
+    extends: ['schedule:earlyMondays']
   },
+
+  // Vulnerability Alerts
+  vulnerabilityAlerts: {
+    enabled: true,
+    addLabels: ['SECURITY'],
+    reviewers: ['team:loopback-maintainers']
+  },
+
+  // Package Rules
   packageRules: [
     {
       automerge: true,
       matchUpdateTypes: [
-        'minor',
-        'patch'
-      ],
+        'patch',
+        'minor'
+      ]
+    },
+    {
+      draftPR: true,
+      matchUpdateTypes: "major"
     },
     {
       automerge: true,
-      matchDepTypes: [
-        'devDependencies'
-      ]
+      matchDepTypes: ['devDependencies']
     },
     {
       enabled: false,
-      matchDepTypes: [
-        'engines'
-      ]
+      matchDepTypes: ['engines']
     },
     {
-      groupName: 'api-extractor packages',
-      matchPackageNames: [
-        '@microsoft/api-extractor',
-        '@microsoft/api-documenter'
-      ]
```

Comment:
Is there a reason why we're removing this group?

Reply:
I have forgotten to incorporate it